met-web/src/config.ts

```diff
@@ -20,24 +20,22 @@ declare global {
     }
 }
 
-const API_URL = window._env_.REACT_APP_API_URL || process.env.REACT_APP_API_URL;
```

Comment:
I'm worried that this would break our existing way of loading environment variables in our DEV/TEST environments

Reply:
you might be right, might need a little help setting something up for the testing then

Reply:
I did check and thats how we load the environment variables, can you somehow pass an empty object to the window._env_ when loading the tests?
e.g.
window['_env_'] = {};

Reply:
we do this in the public/config.js file

Reply:
i added the config.js to the jest setup env so and added the window.env back into the file so everything should be good now :)